src/Compilers/Core/Portable/CommandLine/EditorConfig.SectionNameMatching.cs

```diff
@@ -36,6 +37,25 @@ public static string TryCompileSectionNameToRegEx(string sectionName)
             var sb = new StringBuilder();
             sb.Append('^');
 
+            // EditorConfig matching depends on the whether or not there are
+            // directory separators and where they are located in the section
+            // name. Specifically, the editorconfig core parser says:
+            // https://github.com/editorconfig/editorconfig-core-c/blob/5d3996811e962a717a7d7fdd0a941192382241a7/src/lib/editorconfig.c#L231
+            //
+            //     Pattern would be: /dir/of/editorconfig/file[double_star]/[section] if
```

Comment:
Consider reformatting this comment to put each case on a separate line, rather than each case half on one line and half on another.